build.sbt

```diff
@@ -51,7 +51,7 @@ lazy val root = Project("spark-redshift", file("."))
     // DON't UPGRADE AWS-SDK-JAVA if not compatible with hadoop version
     // https://stackoverflow.com/a/49510602/2544874
     // https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-aws/2.7.7
-    testAWSJavaSDKVersion := sys.props.get("aws.testVersion").getOrElse("1.7.4"),
+    testAWSJavaSDKVersion := sys.props.get("aws.testVersion").getOrElse("1.11.679"),
```

Comment:
Can you make this work without upgrading the AWSJavaSDKVersion? I remember I tried to go to 1.11.xyz but it was not backward compatible. Check comment above for references

Reply:
You'll have to move to a version that supports the new builder. It looks like we have the SDK marked as provided so presumably we're using whatever version is deployed in EMR anyway. It just needs to be a version that implements "AmazonS3ClientBuilder"

Reply:
Thanks for checking! I'll try running the integration test suite internally (I'm only afraid to break our builds) and let you know

Reply:
I completely understand. If someone can help me figure out what's the earliest version to have added that method then we should be able to use it. AWS doesn't make it easy to view old JavaDocs.